net/openssh/files/sshd.init

```diff
@@ -20,19 +111,28 @@ start_service() {
 		}
 	done
 	mkdir -m 0700 -p /var/empty
+	mkdir -m 0700 -p /root/.ssh
 
-	local lport=$(awk '/^Port / { print $2; exit }' /etc/ssh/sshd_config)
```

Comment:
This is needed to read the Port from sshd_config for mdns.
The Port from UCI may be overridden by the sshd_config.
Not so critical but keep in mind.

Reply:
But generally the problem is that old sshd_config may be overridden by the options from the default config that was installed with a new version of a package.

Reply:
Okay, thank you.